d-sgov/d-sgov-vfr-1.8.0-glosář.ttl

```diff
@@ -1730,14 +1730,14 @@ vfr-1.8.0-pojem:má-geometrii-regionu-soudržnosti
 
 vfr-1.8.0:glosář  a     <https://slovník.gov.cz/veřejný-sektor/pojem/glosář> , <http://www.w3.org/2004/02/skos/core#ConceptScheme> , owl:Ontology ;
         rdfs:label      "Slovník výměnného formátu RÚIAN ve verzi 1.8.0 - glosář"@cs , "Vocabulary of the RÚIAN exchange format in version 1.8.0 - glossary"@en ;
-        <http://purl.org/dc/elements/1.1/creator>
+        <http://purl.org/dc/terms/created>
+                "10.12.2019" ;
+        <http://purl.org/dc/terms/creator>
                 "Michal Med" ;
-        <http://purl.org/dc/elements/1.1/rights>
+        <http://purl.org/dc/terms/rights>
```

Comment:
@MichalMed @martinnec Proč je zvolena licence pro nekomerční použití (NC) a zakázány úpravy (ND)?